### PR TITLE
[DT-038467] Work Flow Catalog - Chat with Chat GPT Dialog: Multiple U…

### PR DESCRIPTION
**Pull request by deci-shawns**
Adjusted string output in conversation to format with capital initial in role name, line break after colon, and an additional line break after the message to create a space between chats. Edited conversation form to create padding on sides, realign buttons, and set to vertical stack to prevent validation from being blocked by form elements. Also updated completion form and download forms with padding.

### CODE REVIEW

File: Decisions.OpenAI/Steps/CreateChatCompletion.cs [hunk 1]
```diff
@@ -93,7 +93,7 @@ public ResultData Run(StepStartData data)
             
             foreach (ChatMessage chat in request.Messages)
             {
-                conversation += $"{chat.Role}: {chat.Content}\n";
+                conversation += $"{chat.Role[0].ToString().ToUpper()}{chat.Role.Substring(1)}:\n{chat.Content}\n\n";
```
Comment (jason-decisions) [inline Decisions.OpenAI/Steps/CreateChatCompletion.cs:96]: Could `chat.Role[0]` be null? or `chat.Role` be null?

Comment (deci-shawns) [reply Decisions.OpenAI/Steps/CreateChatCompletion.cs:96]: If it's null, the entire response from OpenAI is null, which would not happen under normal circumstances

Comment (jason-decisions) [reply Decisions.OpenAI/Steps/CreateChatCompletion.cs:96]: We should still check for it though and throw a Business Rule Exception with a readable message in the case that OpenAI returned a null response. Something like "OpenAI did not successfully return a response. Please check the settings and try again."

Comment (deci-shawns) [reply Decisions.OpenAI/Steps/CreateChatCompletion.cs:108]: Added exception

File: Decisions.OpenAI/Steps/CreateChatCompletion.cs [hunk 2]
```diff
@@ -86,14 +87,25 @@ public ResultData Run(StepStartData data)
...
+            if (!string.IsNullOrEmpty(chatResponse.Choices[chatIndex].Message.Role) &&
+                !string.IsNullOrEmpty(chatResponse.Choices[chatIndex].Message.Content))
+            {
+                request.Messages.Add(chatResponse.Choices[chatIndex].Message);
+            }
+            else
+            {
+                throw new BusinessRuleException(
+                    "OpenAI failed to return a chat completion. Please check the settings and try again.");
+            }
```
Comment (jason-decisions) [inline Decisions.OpenAI/Steps/CreateChatCompletion.cs:108]: I still don't think this protects it. You should check `chatResponse`, `chatResponse.Choices`, `chatResponse.Choices[chatIndex]`, `chatResponse.Choices[chatIndex].Message`, 'chatResponse.Choices[chatIndex].Message.Role', and `chatResponse.Choices[chatIndex].Message.Content` for null and throw the exception immediately. Then you don't have to check it for NOT NULL before you use it.

**Review by jason-decisions — APPROVED** [review]

**Pull request merged**